starters/basic-starter/components/meta.tsx

```diff
@@ -0,0 +1,92 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { DrupalMetatag } from 'types/drupal';
+
+interface MetaProps {
+  title?: string;
+  path?: string;
+  tags?: DrupalMetatag[];
+}
+
+export function Meta({ tags }: MetaProps) {
+  const router = useRouter();
+  const origin =
+    typeof window !== 'undefined' && window.location.origin
+      ? window.location.origin
+      : '';
+  const currentUrl = `${origin}${router.asPath !== '/' ? router.asPath : ''}`;
```

Comment:
@lauriii: Since backend domain names have no values, customers can send or skip domain names along with metatags. I believe we should pass only router.asPath to the frontend, which will display relative paths. 

In the case of domain, either we pass it as currently coded or we should pass it through the JS global variable where frontend users can set globally.

Reply:
I like the idea to add an env variable for this. We could call it `NEXT_PUBLIC_SITE_BASE_URL` for example.

Reply:
sounds good @lauriii, we are working on this change.